Packs/EWSMailSender/Integrations/EWSMailSender/EWSMailSender.py

```diff
@@ -195,13 +195,17 @@ def collect_manual_attachments(manualAttachObj):
     return attachments
 
 
+def get_none_empty_addresses(addresses_str):
+    return list(filter(None, addresses_str.split(",")))
```

Comment:
Let's add logs printing of the address_str before the return so in case the fix will not solve the issue you have better logs